hooks/useAvalancheForecastFragment.ts

```diff
@@ -36,6 +38,7 @@ export const useAvalancheForecastFragment = (center_id: string, forecast_zone_id
       });
       throw parseResult.error;
     } else {
+      // TODO(brian): This is assuming that a forecast always exists for the given zone/date range. That's not a good assumption!
```

Comment:
Doesn't the `find` return an option, encoding the possibility of no response in the output type?

Reply:
I think my concern is that we probably want to show the most recent forecast, expired or not. Right now we don't show anything for the Olympics, but there is useful info there, even if it's just to know when the next forecast is coming: https://nwac.us/avalanche-forecast/#/olympics

Reply:
Yeah it is not clear to me what the dates mean on this API - maybe we can reach out to Chris and ask? Are we providing the date range for the publish date for the thing, or the date range for which it's valid. or ... ?

Reply:
Tracked in https://github.com/stevekuznetsov/avalanche-forecast/issues/45